jobs/build/set_client_latest/set-latest.sh

```diff
@@ -18,14 +18,30 @@ TMPDIR=${WORKSPACE}/tools
 mkdir -p "${TMPDIR}"
 cd ${TMPDIR}
 
-# create latest symlink
-ln -svf ${RELEASE} latest
+LESS=$(cat ${RELEASE} |awk -F '.' '{print $1 "." $2 "."  $3 "-0"}')
+ABOVE=$(cat ${RELEASE} |awk -F '.' '{print $1 "." $2+1 "."  $3 "-0"}')
+RELEASE_DIR=$(cat ${RELEASE} |awk -F '.' '{print $1 "." $2}')
+Z_LATEST=$(curl -X GET --fail -G https://openshift-release.svc.ci.openshift.org/api/v1/releasestream/4-stable/latest --data-urlencode "in=>${LESS} <${ABOVE}" |jq  '.name')
```

Comment:
404 can be returned by this query. With `set -e`, this would cause your script to exit.
Using `||` logic will ensure that the error is swallowed and the echo takes precedence to feed jq. You also need `-r` for jq, or it will return something like `"4.2.4"` instead of `4.2.4`. 
```suggestion
$((curl -X GET --fail -G https://openshift-release.svc.ci.openshift.org/api/v1/releasestream/4-stable/latest --data-urlencode "in=>4.3.0-0 <4.4.0-0" || echo '{ "name": "none" }') | jq -r '.name')
```